drivers/tee/optee/core.c

```diff
@@ -354,18 +353,18 @@ optee_config_shm_ioremap(struct device *dev, optee_invoke_fn *invoke_fn,
 	size_t size;
 	phys_addr_t begin;
 	phys_addr_t end;
-	void __iomem *va;
+	void *va;
 	struct tee_shm_pool_mem_info priv_info;
 	struct tee_shm_pool_mem_info dmabuf_info;
 
 	invoke_fn(OPTEE_SMC_GET_SHM_CONFIG, 0, 0, 0, 0, 0, 0, 0, &res.smccc);
 	if (res.result.status != OPTEE_SMC_RETURN_OK) {
-		dev_info(dev, "shm service not available\n");
+		pr_info("shm service not available\n");
```

Comment:
No prefix? (`pr_fmt()`). Won't this be ambiguous to the end-user?
What about making sure that all files using `pr_xxx()` define `pr_fmt()` as:
```
#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
```